### ISSUE TITLE
A11y shrine: missing label

### ISSUE THREAD

**Issue by felixzapata** [PolymerElements/app-layout#359]
### Description

There is no `label` associated with the `select` element.
### Expected outcome

The `select` element should have a `label` element or an `aria-label` attribute.
### Actual outcome

The label is missing.
